components/job-orchestration/job_orchestration/executor/compress/celery_compress.py

```
@@ -1,3 +1,5 @@
+from typing import List
```

Ruff (0.14.1)

`typing.List` is deprecated, use `list` instead

(UP035)